components/Home/Bonding/requests.jsx

```diff
@@ -67,15 +68,40 @@ const getBondingProgramsRequest = async ({ isActive }) => {
  */
 const getProductEventsFn = async (eventName) => {
   const contract = getDepositoryContract();
-
   const provider = getEthersProvider();
   const block = await provider.getBlock('latest');
 
-  const oldestBlock = (getChainId() || 1) >= 100000 ? 50 : 1000000;
```

Comment:
I replaced this with `lookbackBlocks` because:
- it doesn't make sense to me. Why would we lookback a lower range of blocks if the chain id is high?
- the name wasn't very descriptive

Reply:
I believe this is for the forked chainId-s, since the forked ones are very bad with pulling events, and we limit to just 50 last blocks. @mohandast52 please correct me on this one.

Reply:
OK, no worries – will add the comment.